lib/expo/mo/parser.ex

```diff
@@ -96,8 +95,6 @@ defmodule Expo.MO.Parser do
       |> read_table_headers(binary_part(content, start_offset, number_of_elements * 2 * 4), [])
       |> Enum.map(&read_table_cell(content, &1))
 
-  defp read_table_headers(endianness, table_header, acc)
```

Comment:
I find those headers quite useful to get the context / names of the function. Is there any argument against keeping them?

Reply:
Using bodiless clauses for private functions is not conventional in Elixir's stdlib or in Gettext. Since this project is in the elixir-gettext org, I think it's better to stick to the script 😉 I added `_parameter` namings so that it's easier to understand what arguments are.

Reply:
@whatyouhide Is there a place where those conventions are written down?

I have the convention in my team to write function headers with the name if all functions match against a pattern and therefore do not name arguments.

Reply:
@maennchen no, we don't really have a place where we write this stuff down. We usually catch it in code reviews. 🙃